### DISCUSSION TITLE
Slow performance on doc website

### DISCUSSION THREAD

**Discussion by crubier** [heroui-inc/heroui#375] [Feedback]
**Describe the bug**

I was initially very interested by nextui, especially the runtime performance story as compared to chakraui for example. But then I realized the website is noticeably slow. Which is confirmed by profiling:

**Steps to reproduce**

Profil changing pages on various CSS libraries websites:

[ChakraUI](https://chakra-ui.com/docs/form/button) is using emotion, takes ~100ms to change page (which is too slow)

<img width="1476" alt="Screenshot 2022-01-16 at 11 14 50" src="https://user-images.githubusercontent.com/2954572/149657680-64b66f08-4ab2-4f01-bbff-d3537132f455.png">

[Stitches](https://stitches.dev/docs/variants) is much faster, takes ~40ms to change page (better but not zero-runtime as I hoped)

<img width="1477" alt="Screenshot 2022-01-16 at 11 14 39" src="https://user-images.githubusercontent.com/2954572/149657721-13a5d88b-663f-44bd-96da-51edecdc8d3a.png">

But [NextUI](https://nextui.org/docs/components/collapse) takes up to ~500ms to change page (which is very slow, and very noticeable to users)

<img width="1480" alt="Screenshot 2022-01-16 at 11 13 22" src="https://user-images.githubusercontent.com/2954572/149657763-7f58bb92-a93d-4d64-a389-dea2d3300344.png">

**Expected behaviour**

Very fast page change

**Environment / Context**
- Desktop
- Operating System: MacOS
- Browser and version Chrome latest
- Device: Macbook Pro

**Additional info**

Maybe there is a good explanation to why things are this way, maybe there is a low hanging fruit ? I really want to consider nextui.

**Comment by jrgarciadev** [comment]
Hey @crubier, thank you for your detailed explanation, It seems to be affected by having 2 CSS libraries together, we recently migrated all components from `styled-jsx` to `stitches` but the docs website is still running with `styled-jsx` with some parts done in `stitches`, I'm working on a new landing page and on migrating the docs website to `stitches`

I'm glad you want to use NextUI 🙌🏻

**Comment by cguagenti** [comment]
Yup! Noticed the same @crubier.

@jrgarciadev I would love to use NextUI for custom projects. Looks very interesting!

**Comment by iSplasher** [comment]
The docs is still very noticeably slow. Is it still running with 2 css libraries?
